test/functional/apps/discover/group2/_search_on_page_load.ts

```diff
@@ -144,7 +145,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await testSubjects.click('discoverNewButton');
         await PageObjects.header.waitUntilLoadingHasFinished();
 
-        await retry.waitFor('number of fetches to be 1', waitForFetches(1));
+        await retry.waitFor('number of fetches to be 0', waitForFetches(0));
```

Comment:
actually, this is now correct since we set the page to "loading" state when saved searches are changed